Fix indentation when using .append method

nlemoine opened the pull request:
Hi @dotnil 

One last minor issue with the `.append` method. When using it, the ssh config will look like:
```
Host tahoe
    HostName tahoe.com
    Host walden
    HostName waldenlake.org
```
Instead of:
```
Host tahoe
    HostName tahoe.com

Host walden
  HostName waldenlake.org
```

While it still works that way, adding a new line at the end of each section makes it more readable.

nlemoine commented:
Also added recent tags to the right commits. Don't know if that can be part of the PR.

dotnil commented:
Ah you got me. The indentation part of `.append()` is quite rough. I'll see to this pr asap.

dotnil commented:
@nlemoine quick question, do you mostly indent ssh config with 2 or 4 spaces? I've noticed the indentation isn't consistent in the pr description.

dotnil commented on index.js:
```diff
@@ -91,6 +91,7 @@ class SSHConfig extends Array {
   append(opts) {
     let config = this
     let configWas = this
+    const lastParam = opts[Object.keys(opts)[Object.keys(opts).length - 1]]
```
`opts[Object.keys(opts).pop()]` would be better. But I believe we don't need to do lastParam check here.

dotnil replied:
To add an extra line break at the bottom of the appended config, we can just do a `config[config.length - 1].after += '\n'` at the rear of this function.

dotnil commented:
ping @nlemoine

nlemoine commented:
> do you mostly indent ssh config with 2 or 4 spaces?

Sorry, that's just a typo mistake. 

Thanks for the review. I'll make the changes you requested.

nlemoine commented:
Just added your suggested change. 
Also added a tab before "sub" config lines to increase readability so the config will look like: 
```
Host tahoe
	HostName tahoe.com
	Port 1234
```

nlemoine commented:
Finally added test.

nlemoine commented:
Just noticed you're using two spaces. The last commit fixes this. 

BTW, in your `parse` method, the 2 spaces end up in the `after` property of the previous line object. It would make more sense to use the `before` property as the object is representing a line.

dotnil commented:
Good point, I'll make the change after this pr.

dotnil commented:
v1.1.3 released

nlemoine commented:
Thanks! 👍
